### ISSUE TITLE
Error while connecting to WPA Enterprise network

### ISSUE THREAD

**Issue by kbshal** [martin-ger/esp_wifi_repeater#509]
It gives an error while trying to connect to the enterprise network

**Comment by vakshit** [comment]
how do you check for errors?
i'm having the same issue. I'm able to connect to a WPA2 network but not able to connect to enterprise network.

**Comment by martin-ger** [comment]
ESP8266 and WPA Enterprise  is a pain. Think, often the certificates used by the WPA Enterprise networks are too large for the ESP8266's memory and this causes crashes. If possible, try with the ESP32 instead.

**Comment by vakshit** [comment]
my enterprise network doesn't use CA certificates. if that's what you are talking about.

**Comment by martin-ger** [comment]
What kind of authentication do you use?

**Comment by vakshit** [comment]
![image](https://user-images.githubusercontent.com/82945402/213548694-168a8ec0-681f-4244-a39a-ad92b6622330.png)

this config after entering my username and password works well on my system.

**Comment by martin-ger** [comment]
This only means, this config doesn't check the server's certificate with a CA cert - there is the chance that you connect to a fake AP. Still the RADIUS server has a cert and sends it. See also: https://security.stackexchange.com/questions/182530/is-it-possible-for-an-802-1x-network-peap-mschapv2-to-have-no-certificate

**Comment by martin-ger** [comment]
MSCHAPv2 itself isn't a problem here as it is used inside a TLS tunnel to check the PW.

**Comment by vakshit** [comment]
Yes, there is a chance that the RADUIS server sends me a CA certificate, but i dont know a way to check it. Also, it is my institute's wifi so it's not getting connected to a fake AP. 

Shall I try on an ESP32? I'm actually still unable to find out the issue. 😅

**Comment by martin-ger** [comment]
Think, you should try.

**Comment by vakshit** [comment]
the same code or this one [https://github.com/martin-ger/esp32_nat_router](https://github.com/martin-ger/esp32_nat_router).

**Comment by abinthomas744** [comment]
> how do you check for errors? i'm having the same issue. I'm able to connect to a WPA2 network but not able to connect to enterprise network.

same question, I'm looking for a regularly flashing light to indicate an uplink but can't see it other than that no errors as such
